Resources/Prototypes/Entities/Structures/Windows/window.yml

```diff
@@ -64,7 +64,13 @@
   - type: IconSmooth
     key: windows
     base: window
-  - type: Window
+  - type: Tag
+    tag: Window
```

Comment:
Ah other problem, window already has tag component. its the first one in the list.

Reply:
Caught that just now while figuring out how to read the Linter. Thank you though!

Reply:
Also, any component that inherits from window that defines its own tags comp (e.g. directional windows n such) will also need to be given the tag

Reply:
You're right, I did miss one instance of type: Window in the yml!

Reply:
Oh sorry I commenting without refreshing the page and missed those comments.

Reply:
I too was commenting without refreshing just now! And pushing commits real quickly. You're good!